app/views/layouts/application.haml

```diff
@@ -45,11 +45,11 @@
             - sidebar_menu_items.each do |key|
               = render('layouts/sidebar_menu_item', path_key: key)
     %section.content#content
-      - if Rails.configuration.x.top_menu_items.present?
-        %div.submenu
-          %nav.main-submenu.visible
-            %a.submenu-item.selected{ href: '#' }= t('menu.deploy')
-            = top_menu_items(@outputs)
+      %div.submenu
+        %nav.main-submenu.visible
+          %a.submenu-item.deploy{ :href  => "/welcome"}= t('menu.deploy')
```

Comment:
I think we need to remove the `deploy` entry and make the other 2 disabled initially

Reply:
why we should remove the `deploy` here? this layout is used as central place.

Removing the `deploy` would affect the whole UX.

How do you means to make the others 2 disabled? I'm not still familiar with blue-horizion javascript/css since I joned the UI really recenlty

Reply:
Removing the "deploy" link is one of the points of Ken's feedback: there is no point in providing access a section that cannot be accessed anymore.

Reply:
Ok removed